internal/pkg/controller/controller.go

```diff
@@ -175,7 +183,11 @@ func (c *Controller) Run(threadiness int, stopCh chan struct{}) {
 
 func (c *Controller) runWorker() {
 	// At this point the controller is fully initialized and we can start processing the resources
-	controllerInitialized = true
+	if c.resource == "secrets" {
```

Comment:
do you think it would be nicer to use a switch case statement?

    switch c.resource {
    case
        "secrets",
        "configMaps",
        return true
    }
    return false